public/linkedhashes.php

```diff
@@ -67,7 +67,7 @@
 
             echo '<br/><code> ' . $hash['Hash'] . '</code>';
             if (!empty($hash['User'])) {
-                echo ' linked by ' . GetUserAndTooltipDiv($hash['User']);
+                echo ' linked by ' . userAvatar($hash['User']);
```

Comment:
this didn't used to have an image:
![image](https://user-images.githubusercontent.com/32680403/192919232-a2566374-9287-493b-8d62-4cc66edaeadf.png)

or possible line 84 (or both)

Reply:
Adjusted accordingly